src/components/ui/Button.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { ButtonHTMLAttributes, ReactNode } from "react";
-import { ButtonSize, ButtonVariant } from "../../utils/types";
+import React, { ButtonHTMLAttributes, ReactNode } from 'react'
+import { ButtonSize, ButtonVariant } from '../../utils/types'
 
 interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-  children: ReactNode;
-  variant?: ButtonVariant;
-  outlined?: boolean;
-  size?: ButtonSize;
+  children: ReactNode
+  variant?: ButtonVariant
+  outlined?: boolean
+  size?: ButtonSize
+  className?: string
```

Comment:
We’re not allowing class names in the button component to avoid accidental design changes that could lead to inconsistency. Let’s stick to the current structure to keep everything looking uniform!